File: EveVoid/ClientApp/src/app/api/models/main-character-dto.ts
/* tslint:disable */
import { EsiCharacterDto } from './esi-character-dto';
import { MapLayoutDto } from './map-layout-dto';
import { MaskType } from './mask-type';
export interface MainCharacterDto {
  pilotId?: number;
  allianceId?: number;
  allianceName?: string;
  corporationId?: number;
  corporationName?: string;
  esiCharacters?: Array<EsiCharacterDto>;
  id?: number;
  mapLayouts?: Array<MapLayoutDto>;
  maskType?: MaskType;
  pilotName?: string;
}

File: EveVoid/Dto/MainCharacterDto.cs
using EveVoid.Models.Navigation.Masks;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EveVoid.Dto
{
    public class MainCharacterDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int? CorporationId { get; set; }
        public string CorporationName { get; set; }
        public int? AllianceId { get; set; }
        public string AllianceName { get; set; }
        public MaskType MaskType { get; set; }

        public List<EsiCharacterDto> EsiCharacters { get; set; }
        public List<MapLayoutDto> MapLayouts { get; set; }

        public MainCharacterDto()
        {
            EsiCharacters = new List<EsiCharacterDto>();
            MapLayouts = new List<MapLayoutDto>();
        }
    }
}
